### ISSUE TITLE
False positive lottie-web on angular projects 

### ISSUE THREAD

**Issue by segux** [depcheck/depcheck#869]
Hi, im trying to use `lottie-web`  dependency and is getting false positive on unused dependenciees

It's a dependency to load lotties json and render into html

```javascript
{
  "dependencies": [
    "@angular/animations",
    "@angular/common",
    "@angular/compiler",
    "@angular/core",
    "@angular/forms",
    "@angular/platform-browser",
    "@angular/router",
    "lottie-web",
    "rxjs",
    "tslib",
    "zone.js"
  ],
  "devDependencies": [
    "@angular-devkit/build-angular",
    "@angular/compiler-cli",
    "@types/jasmine",
    "@types/node",
    "codelyzer",
    "jasmine-core",
    "jasmine-spec-reporter",
    "karma",
    "karma-chrome-launcher",
    "karma-coverage",
    "karma-jasmine",
    "karma-jasmine-html-reporter",
    "protractor",
    "ts-node",
    "tslint",
    "typescript"
  ],
  "missing": {},
  "using": {
    "@angular/cli": ["/home/segux/depcheck-false-positive/package.json"]
  },
  "invalidFiles": {
    "/home/segux/depcheck-false-positive/tsconfig.json": "SyntaxError: /home/segux/depcheck-false-positive/tsconfig.json: Unexpected token '/', \"/* To lear\"... is not valid JSON\n    at parse (<anonymous>)\n    at Module._extensions..json (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:55:15733)\n    at Module.load (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:55:13457)\n    at Module._load (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:55:10535)\n    at Module.require (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:55:13775)\n    at i (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:98:2198)\n    at _0xd5385f (https://depcheckfalsepositive-03ry.w-credentialless.staticblitz.com/blitz.a2aabdd9.js:352:196402)\n    at readJSON (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/depcheck/dist/utils/index.js:24:10)\n    at parseTTypeScript (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/depcheck/dist/special/ttypescript.js:16:41)\n    at getDependencies (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/depcheck/dist/check.js:119:24)",
    "/home/segux/depcheck-false-positive/src/main.ts": "SyntaxError: Unexpected character '´'. (27:6)\n    at constructor (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:356:19)\n    at TypeScriptParserMixin.raise (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:3223:19)\n    at TypeScriptParserMixin.getTokenFromCode (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:2885:16)\n    at TypeScriptParserMixin.getTokenFromCode (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:6965:11)\n    at TypeScriptParserMixin.getTokenFromCode (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:9847:11)\n    at TypeScriptParserMixin.nextToken (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:2347:10)\n    at TypeScriptParserMixin.next (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:2258:10)\n    at TypeScriptParserMixin.eat (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:2262:12)\n    at TypeScriptParserMixin.expect (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:3585:10)\n    at TypeScriptParserMixin.parseObjectLike (file:///home/.npm/_npx/d2bd3cb5801d0b99/node_modules/@babel/parser/lib/index.js:11650:14)"
  },
  "invalidDirs": {}
}
``` 

Example to reproduce the error: 
https://stackblitz.com/~/github.com/segux/depcheck-false-positive
